src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -413,25 +412,21 @@ private static boolean checkForTutorialMap() {
       return false;
     }
 
-    final StringBuilder messageBuilder = new StringBuilder();
-    messageBuilder.append("<html>");
-    messageBuilder.append("Would you like to download the tutorial map?<br>");
-    messageBuilder.append("<br>");
-    messageBuilder.append("(You can always download it later using the Download Maps<br>");
-    messageBuilder.append("command if you don't want to do it now.)");
-    messageBuilder.append("</html>");
-    SwingComponents.promptUser("Welcome to TripleA", messageBuilder.toString(), () -> {
+    String message = "<html>" +
+            "Would you like to download the tutorial map?<br>" +
+            "<br>" +
+            "(You can always download it later using the Download Maps<br>" +
+            "command if you don't want to do it now.)" +
+            "</html>";
```

Comment:
Multiple Checkstyle violations within this statement: "'+' should be on a new line."

Also, `message` can be final.

Reply:
Alternatively this string could continue to be built using a builder, but making use of the propertied of a builder instead:
```
final String message = new StringBuilder()
.append("html")
.append()
.
.
.toString();
```

Reply:
Java compilers will convert string concatenation to a string builder automatically. There is no need to use a string builder here.